Crosslink SQL Performance pages (#1827)
OctopusDeploy/docs: docs/administration/data/octopus-database/index.md

## Routine maintenance {#maintenance}

> You are responsible for the routine maintenance of your Octopus database. Performance problems with your SQL Server will make Octopus run and feel slow and sluggish. You should implement a routine maintenance plan for your Octopus database. Here is a [sure guide](https://oc.to/SQLServerMaintenanceGuide) (free e-book) for maintaining SQL Server. We have some generic recommendations in our [Performance](https://octopus.com/docs/administration/managing-infrastructure/performance#sql-maintenance) section that may help get you started.

Reviewer: ```suggestion
You are responsible for the routine maintenance of your Octopus database. Performance problems with your SQL Server will make Octopus run and feel slow and sluggish. You should implement a routine maintenance plan for your Octopus database. Here is a [sure guide](https://oc.to/SQLServerMaintenanceGuide) (free e-book) for maintaining SQL Server. Our [Performance](/docs/administration/managing-infrastructure/performance/index.md#sql-maintenance) section has some general recommendations that may help get you started.
```
